src/parsers/perf/perfparser.cpp

```diff
@@ -1405,7 +1407,14 @@ void PerfParser::startParseFile(const QString& path, const QString& sysroot, con
         }
 
         QProcess process;
-        process.setProcessEnvironment(Util::appImageEnvironment());
+        auto env = Util::appImageEnvironment();
+
+        if (!Settings::instance()->debuginfodUrls().isEmpty()) {
+            env.insert(QStringLiteral("DEBUGINFOD_URLS"),
```

Comment:
we shouldn't override any existing urls, but just prepend our custom urls on top, i.e.if hotspot is started with `DEBUGINFOD_URLS` already set to something

Reply:
done, tested by adding an invalid url in hotspot and defining a working one in DEBUGINFOD_URLS